content/en/tests/setup/ruby.md

```diff
@@ -79,6 +94,127 @@ gem "datadog-ci", "~> 1.0", group: :test
 
 ## Instrumenting your tests
 
+Follow these steps if your CI Provider is not supported for auto-instrumentation (see [Configuring reporting method](#configuring-reporting-method)).
+
+1. Set the following environment variables to configure the tracer:
+
+`DD_CIVISIBILITY_ENABLED=true` (Required)
+: Enables the Test Optimization product.
+
+`DD_ENV` (Required)
+: Environment where the tests are being run (for example: `local` when running tests on a developer workstation or `ci` when running them on a CI provider).
+
+`DD_SERVICE` (Required)
```

Comment:
As we're transitioning to the repo-centric view, I think it makes sense to mark the service name as optional

Reply:
true